src/main/resources/reference.conf

```diff
@@ -3,5 +3,7 @@ kamon {
     // if enabled kamon will set MDC context with TraceID and SpanID valuse
     // they could be referenced using kamonTraceID and kamonSpanID keys
     mdc-context-propagation = on
+    mdc-trace-key = kamonTraceId
```

Comment:
I think these settings should be called `mdc-trace-id-key` and `mdc-span-id-key` to be very very specific on what ends up getting there.

Reply:
I wanted to be consistent with `AsyncAppenderInstrumentation.MdcSpanKey` naming, which was already there. Maybe those should be renamed as well?

Reply:
This whole topic made me think that the settings names should be changed a bit, probably under a `mdc` section and with one more feature we have in mind (copying arbitrary context keys into the MDC). 

For now I would say, change it for the sake of consistency, but it will most likely change in a upcoming PR. I'll ping both of you there for feedback! :D